src/main/scala/cromwell/engine/db/DataAccess.scala

```diff
@@ -2,13 +2,14 @@ package cromwell.engine.db
 
 import cromwell.binding.values.WdlValue
 import cromwell.binding.{Call, FullyQualifiedName, WdlJson, WdlSource}
-import cromwell.engine.ExecutionStatus.ExecutionStatus
 import cromwell.engine.backend.Backend
-import cromwell.engine.{SymbolStoreKey, SymbolStoreEntry, WorkflowId, WorkflowState}
+import cromwell.engine.{SymbolStoreEntry, WorkflowId, WorkflowState}
 
 import scala.concurrent.Future
 
 object DataAccess {
+  def apply() = new slick.SlickDataAccess()
```

Comment:
The reason was because there was more than one caller for the same code. If in the future we need to change backends (Silver team was moving to switch to OrientDB from Slick), all DataAccess clients (server & command line) can be swapped with this one line change.

I did fix the return type though to be `DataAccess` instead of `SlickDataAccess`.